prefect_hex/project.py

```diff
@@ -52,29 +52,18 @@ async def run_project(
     Returns:
         Information about the triggered project run.
     """  # noqa
-    endpoint = "/project/{project_id}/run"  # noqa
-
-    responses = {}
-
-    params = {
-        "project_id": project_id,
-    }
-
-    json_payload = {
-        "input_params": input_params,
-        "dry_run": dry_run,
-        "update_cache": update_cache,
-    }
+    endpoint = f"/project/{project_id}/run"  # noqa
 
     response = await execute_endpoint.fn(
         endpoint,
         hex_credentials,
         http_method=HTTPMethod.POST,
-        params=params,
-        json=json_payload,
+        json=models.RunProjectRequestBody(
+            dryRun=dry_run, inputParams=input_params, updateCache=update_cache
```

Comment:
Does using snake case like`dry_run=dry_run` work?

Reply:
No because this model has aliases to match up with the format of the Hex API.